ini.c

```diff
@@ -202,7 +202,12 @@ int ini_parse_stream(ini_reader reader, void* stream, ini_handler handler,
             }
             else if (!error) {
                 /* No '=' or ':' found on name[=:]value line */
-                error = lineno;
+#if INI_ALLOW_KEY_WITHOUT_EQUAL
+                *end = '\0';
+                name = rstrip(start);
+                if (!HANDLER(user, section, name, NULL))
+#endif
+			error = lineno;
```

Comment:
This is clever :-) but the indentation not matching up is a bit weird. Can we do this instead?

```c
#if INI_ALLOW_NO_VALUE
        *end = '\0';
        name = rstrip(start);
        if (!HANDLER(user, section, name, NULL))
            error = lineno;
#else
        error = lineno;
#endif
```